### DISCUSSION TITLE
500 included, then $10 per 1000 VS Firebase  200,000

### DISCUSSION THREAD

**Discussion by nokia6290** [supabase/supabase#18578] [Questions]
Hi,

Deciding between Firebase VS Supabase.

Concurrent Peak Connections between them is astronomical.

Firebase allows 200 000 while Supabase 500

What's the catch? Is that how Supabase makes money like a hidden fee?

Thx

**Comment by GaryAustin1** [comment]
Not sure how it is a hidden fee when they told you what it is.  Both companies have radically different pricing models for realtime.
You don't mention Firebase is $5GB for data stored and $1GB for data downloaded after free limit.  Your realtime data must be in a different database also.

Supabase is $0.09GB download after your 250GB included. and $0.125GB for data stored which is the same data you can access with database commands.

So depending what you are doing that can make a huge difference with either method.   Without doing the math seems like Supabase is better priced for realtimewith fewer users to lots of data but Firebase is priced better if you have lots of users and a small amount of data.

**Reply by nokia6290** [reply]
My topic was about **Concurrent Peak Connections**.
The 500 limit is just a joke. Making a big deal to skip supabase completely. I'm aware that supabase is experimenting with their prices and this variable loses against competition.

I hope they will improve this number in the future.

**Comment by dannyhchan** [comment]
I'm looking at the Pro plan and wanted to confirm that $10 for an additional 1000 concurrent active connections.  So basically, based on the monthly cost of $25/month + $10, I can have 1500 active concurrent connections correct?

**Reply by GaryAustin1** [reply]
Yes... but you also need to consider if your compute/memory can handle it...
https://supabase.com/docs/guides/realtime/postgres-changes#limitations
